src/main/java/Square.java

```diff
@@ -0,0 +1,11 @@
+public class Square extends Rectangle {
+    double  WIDTH = this.LENGTH;
```

Comment:
to nie są stałe, a zmienne. :) 
Zmienne piszemy tak: nazwaZmiennej

Stałe piszemy tak: NAZWA_ZMIENNEJ

jak odróżnić zmienną od stałej?

Stała:
```
private static final PLN_CURRENCY = 'PLN'
```

Zmienna:
```
private finale String currency;

new Construtor(String currency){
   this.currency = currency // jest to uzupelniane w konstruktorze, czasami może być to 'PLN' czasami 'USD'
}
```

Reply:
więc wszystkie te WIDTH/LENGTH byly akruat dobrze - powinno byc width/length, bo to są pola, a nie stałe. W zależności od tego jaki obiekt stworzysz to bedą się one rózniły. (np. będziesz mial Square o width 3, innych query o width 5 itd)

Gdyby to byla stała to wtedy zawsze to by był Square o width 3 i nigdy nie byłbyś w stanie stworzyć innego.

Reply:
Inny przykład stalej - PI. 
Bo przecież PI to zawsze 3,14.....
 :)
Nigdy PI nie będzie inną liczbą niż 3,14....